### DISCUSSION TITLE
Tiempos para estudiar el material.

### DISCUSSION THREAD

**Discussion by Lyraes** [Launch-X-Latam/MisionFrontEnd#6] [Q&A]
Hay alguna restricción respecto al progreso entre las unidades? Puedo avanzar lo que pueda antes de las siguientes sesiones?

**Comment by akock** [comment]
el comander indico que el ira desbloqueando según avance el curso

**Comment by DiegoHC06** [comment]
Todos la semana va a estar explicando los temas, de igual manera puedes visualizar si está libre los temas antes e ir estudiando lo y cuando esté live, puedes realizar las preguntas, pero mejor esperar que nos indique el mission commander. Así puedes ir paso a paso sin perder el avance. 

Saludos y  mucho éxito

**Comment by JosafatJimenezB** [comment]
Para la cuestión de los temas estos los va a ir explicando por semana, toda esta semana será como una intro a lo web, la siguiente a html y así hasta terminar los temas, las practicas por lo regular se entregan los fines de semana

**Comment by iam-leo** [comment]
Yo recomiendo que lean todo lo que más puedan del módulo correspondiente antes de cada live, si surgen dudas o no entienden determinados términos hagan preguntas en la comunidad o complementen investigando por su propia cuenta (esto les ayudará a formarse como autodidactas) y si aún así siguen sin comprender pues apunten sus preguntas y luego las hacen en los lives. Al menos así van a llegar mejor preparados al día del live y les resultará más fácil comprender lo que explique nuestro Mission Commander @romarpla :rocket:

**Reply by AlvaroFierro** [reply]
Asi es como comenta @iam-leo es preferible ya repasar de antemano el contenido antes de clase, para que mas que nada sean de dudas, recuerden que es contenido autodidacta. De hecho en la certificacion de AZ SC900 eso nos decia @romarpla 

Ademas les recomiendo revisar por su cuenta algunos roadmaps como [roadmap.sh](https://roadmap.sh/) o [freecodecamp](https://www.freecodecamp.org/espanol/learn/) 

Al igual que le empiezen dar mas al ingles, para poder leer documentacion oficial (la [documentacion oficial de JavaScript](https://developer.mozilla.org/es/) es Mozilla)
